Doc/library/multiprocessing.rst

```diff
@@ -878,6 +878,12 @@ For an example of the usage of queues for interprocess communication see
 
    It is a simplified :class:`Queue` type, very close to a locked :class:`Pipe`.
 
+   .. method:: close()
+
+      Close the queue.
```

Comment:
From the reader's point of view, it isn't clear what the point of the method is. Can you add a sentence explaining what the consequences are? (I suppose that the underlying resources are released and the queue becomes unusable?)

Reply:
> (I suppose that the underlying resources are released and the queue becomes unusable?)

Right. I completed the documentation. Does it look better?